asyncapi/twitter/Ballerina.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+license = ["Apache-2.0"]
+keywords = ["Cost/Freemium" , "Vendor/Twitter"]
```

Comment:
Let's add the missing keyword "Trigger" here

Reply:
Added that Keyword